applications/FemToDemApplication/python_scripts/MainCouplingPfemFemDem.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import print_function, absolute_import, division  #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics as KM
+import KratosMultiphysics.FemToDemApplication as FEMDEM
+import KratosMultiphysics.PfemFluidDynamicsApplication as PFEM
+import KratosMultiphysics.FemToDemApplication.MainCouplingFemDem    as MainCouplingFemDem
+import KratosMultiphysics.FemToDemApplication.MainPFEM_for_coupling as MainPFEM_for_coupling
+
+def Wait():
+    input("PFEM-FEMDEM -> Press Something")
+
+def KratosPrintInfo(message):
+    """This function prints info on screen
+    """
+    KM.Logger.Print("", message)
+    KM.Logger.Flush()
+#============================================================================================================================
+class MainCouplingPfemFemDem_Solution:
+#============================================================================================================================
+
+    def __init__(self, Model, PFEMparameters):
+        # Initialize solutions of the FEMDEM and PFEM
+        self.FEMDEM_Solution = MainCouplingFemDem.MainCoupledFemDem_Solution(Model)
+        self.FEMDEM_Solution.Initialize()
+
+        self.PFEM_Solution = MainPFEM_for_coupling.MainPFEM_for_coupling_solution(Model, 
+                                                                                  self.FEMDEM_Solution.FEM_Solution.main_model_part,
+                                                                                  PFEMparameters)
+        KratosPrintInfo("    ___                  _          _            _ _   _         ___  ___  __       "  + "\n" +
```

Comment:
```suggestion
        KratosPrintInfo("","    ___                  _          _            _ _   _         ___  ___  __       "  + "\n" +
```

Reply:
Remember to put at least an empty label

Reply:
This is done inside the KratosPrintInfo function :)